src/Compilers/Server/VBCSCompilerTests/CompilerServerTests.cs

```diff
@@ -329,10 +359,11 @@ private void VerifyResultAndOutput(ProcessResult result, TempDirectory path, str
             }
         }
 
-        [ConditionalFact(typeof(DesktopOnly))]
+        [Fact(Skip = "CompilerServerExecutable no longer exists for this test")]
         [WorkItem(946954, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/946954")]
         public void CompilerBinariesAreNotX86()
         {
+            string CompilerServerExecutable = null;
             Assert.NotEqual(ProcessorArchitecture.X86,
                 AssemblyName.GetAssemblyName(CompilerServerExecutable).ProcessorArchitecture);
         }
```

Comment:
Could this be a `DesktopOnly` test?

Reply:
Probably, yeah. It might be a little hacky but I should be able to get it to work.

Reply:
Turned out to be not so bad!